docker/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN ${CONDA_PREFIX}/bin/conda init bash
 COPY --chown=compass_user:compass_user . /home/compass_user/OPERA/COMPASS
 
 # create CONDA environment
-RUN conda create --name "COMPASS" --file /home/compass_user/OPERA/COMPASS/docker/specifile.txt
+RUN conda create --name "COMPASS" --file /home/compass_user/OPERA/COMPASS/docker/specifile.txt -c conda-forge --override-channels
```

Comment:
A question: It seems all dependencies in `specifile.txt` has `conda-forge` in their URLs (correct me if I missed something). Are those options are really necessary, or just to make sure that all packages are from conda forge?

Reply:
Deferring this question to @LiangJYu who provided the `specifile.txt`

Reply:
Good catch @seongsujeong. You're correct what I added here is unnecessary. I got the specifile.txt confused with requirements.txt.

Reply:
@LiangJYu will you take care of this?